File: Assignment2/program3.c
////////////////////////////////////////////////////////////////
//
//  File Name   : program3.c
//  Description : Accepts a Number, checks it if it's less then 10 prints "Hello" else prints "Demo" on Consol
//  Author      : Akhilesh.P.Sonavane
//  Date        : 10/05/2025
//
////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////
// 
//  Function Name : Display
//  Description   : Checks the inputed Integer is less than 10 and then prints "Hello" else prints "Demo" 
//  Input         : Integer
//  Output        : String
//
////////////////////////////////////////////////////////////////

void Display(int iNo)
{
    if(iNo < 10)
    {
        printf("Hello\n");
    }
    else 
    {
        printf("Demo\n");
    }
}

int main()
{
    int iValue = 0;
    printf("Enter Number\n");
    scanf("%d",&iValue);

    Display(iValue);
    return 0;
}

File: Assignment19/program3.java
////////////////////////////////////////////////////////////////
//
//  File Name   : program3.java
//  Description : Accept a N Integer and Checks if 11 is Present in Elements 
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 22/05/2025
//
////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////
// 
//  Class Name    : Display
//  Function Name : Check
//  Description   : Inputs N Integer and Checks if 11 is Present in Elements 
//  Input         : Integer
//  Output        : Boolean
//
////////////////////////////////////////////////////////////////

class Display
{
    public boolean Check(int[] iPtr, int iSize)
    {
        int iCnt = 0;
        boolean bResult = false;
     
        for(iCnt = 0; iCnt < iSize; iCnt++)
        {
            if(iPtr[iCnt] == 11)
            {
                bResult = true;
                break;
            }
        }

        return bResult;
    }
}
public class program3
{
    public static void main(String args[])
    {
        int[] iArr;
        int iLength = 0;
        int iCnt = 0;
        boolean bRet = false;
    
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number of Elements : ");
        iLength = sobj.nextInt();

        iArr = new int[iLength];

        for(iCnt = 0; iCnt < iLength; iCnt++)
        {
            System.out.print("Element "+(iCnt+1)+" : ");
            iArr[iCnt] = sobj.nextInt();
        }

        Display dobj = new Display();

        bRet = dobj.Check(iArr, iLength);

        if(bRet == true)
        {
            System.out.println("11 is Present");
        }
        else
        {
            System.out.println("11 is Absent");
        }

        sobj.close();
    }
} 